packages/application-shell/src/components/authenticated/authenticated.tsx

```diff
@@ -10,7 +10,7 @@ declare let window: ApplicationWindow;
 
 type RenderFnArgs = { isAuthenticated: boolean };
 export type TAuthenticatedProps = {
-  render: (args: RenderFnArgs) => JSX.Element;
+  render: (args: RenderFnArgs) => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.